### ISSUE TITLE
Installing .NET 8.0.0 issue

### ISSUE THREAD

**Issue by BohdanPodkolzin** [dotnet/core#9019]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/create
Operating System: Windows

Have an issue with downloading .NET SDKs, I reinstalled it a few times (check the IMGs)
I even tried to install the latest .NET from https://dotnet.microsoft.com/en-us/download/visual-studio-sdks
And it is still not working. What is the problem it could be?

![image](https://github.com/dotnet/core/assets/145167186/a0d3b6fb-cfa3-42e1-9301-d64d123677ec)
![image](https://github.com/dotnet/core/assets/145167186/3fd7e1f8-a153-4183-8d24-57e58552fc7c)
![image](https://github.com/dotnet/core/assets/145167186/2efb5d1b-abdc-445d-9493-507639708bb9)

**Comment by BohdanPodkolzin** [comment]
If something went wrong with x64 or armx64 version, try to download x86 version. It helps me

**Comment by mairaw** [comment]
That makes sense since your architecture is x86. I'm curious if the page didn't show correctly that you should download x86?

https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/install

On my case, I have a x64 machine and the link to download is x64:
![image](https://github.com/dotnet/core/assets/12971179/0f5d4d51-d08a-414d-83e5-c3b02ef81992)

what do you see in your machine?

**Comment by BohdanPodkolzin** [comment]
![image](https://github.com/dotnet/core/assets/145167186/cc27cada-bda9-4160-9b98-b1135dc07e04)
@mairaw, check this out. And that was a problem. I reinstalled x64 version like 20 times. Can`t found out where is mistake, but in the end i recognize all staff i have in x86 version. Funny

**Comment by mairaw** [comment]
Reopening this so our team can investigate this more. We might need some additional info from you.
What Windows version are you running?

**Comment by BohdanPodkolzin** [comment]
> Reopening this so our team can investigate this more. The page should be displaying your architecture and not x64. So I’d like to try to fix the bug. We might need some additional info from you. What Windows version are you running? And which browser are you using?

Windows 10, version 21H2. Google Chrome. The latest update

**Comment by mairaw** [comment]
Can you confirm to me that your system is really x86? maybe it's just a PATH issue.

First, confirm your architecture. You can check on the about page of Windows settings or type this command
`wmic OS get OSArchitecture`

If it returns x64/64-bit, then you should check what `where.exe dotnet` returns.

If x86 is listed first and your PC is x64, then you need to follow these instructions:
https://github.com/dotnet/core/issues/5962#issuecomment-780084535

**Comment by BohdanPodkolzin** [comment]
@mairaw check this out

![image](https://github.com/dotnet/core/assets/145167186/752488b0-852d-4130-ab42-642ff0623141)

**Comment by mairaw** [comment]
Yeah, so now please check what `where.exe dotnet` returns. That result tells me that probably your system is x64 but the path is pointing to x86 first, causing these issues.

If x86 is listed first, follow the instructions listed at https://github.com/dotnet/core/issues/5962#issuecomment-780084535
